test/libsolidity/GasCosts.cpp

```diff
@@ -96,7 +96,7 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	compileAndRun(sourceCode);
 
 	if (Options::get().evmVersion() <= EVMVersion::byzantium())
-		CHECK_DEPLOY_GAS(134209, 130895);
+		CHECK_DEPLOY_GAS(134145, 130895);
```

Comment:
Why did these gas values change?

Reply:
Unfortunately, I don't know! I changed these in order to let the tests pass. Do you know why the version bump could have decreased deploy costs? I took the cbor version into consideration, but that should be a fixed amount of bytes that would not change with a version, or would it?

Reply:
Ah right, it would depend on the amount of 0's I guess